hassio/src/addon-view/hassio-addon-info.ts

```diff
@@ -0,0 +1,833 @@
+                    ></ha-switch>
+                  </div>
+                  ${this.addon.ingress
+                    ? html`
+                        <div class="state">
+                          <div>Show in sidebar</div>
+                          <ha-switch
+                            @change=${this._panelToggled}
+                            .checked=${this.addon.ingress_panel}
+                            .disabled=${this._computeCannotIngressSidebar}
+                          ></ha-switch>
+                          ${this._computeCannotIngressSidebar
+                            ? html`
+                                <span>
+                                  This option requires Home Assistant 0.92 or
+                                  later.
+                                </span>
+                              `
+                            : ""}
+                        </div>
+                      `
+                    : ""}
+                  ${this._computeUsesProtectedOptions
+                    ? html`
+                        <div class="state">
+                          <div>
+                            Protection mode
+                            <span>
+                              <iron-icon icon="hassio:information"></iron-icon>
+                              <paper-tooltip>
+                                Grant the add-on elevated system access.
+                              </paper-tooltip>
+                            </span>
+                          </div>
+                          <ha-switch
+                            @change=${this._protectionToggled}
+                            .checked=${this.addon.protected}
+                          ></ha-switch>
+                        </div>
+                      `
+                    : ""}
+                `
+              : ""
+          }
+          ${
+            this.error
+              ? html`
+                  <div class="errors">${this.error}</div>
+                `
+              : ""
+          }
+        </div>
+        <div class="card-actions">
+        ${
+          this.addon.version
+            ? html`
+                <mwc-button class="warning" @click=${this._uninstallClicked}>
+                  Uninstall
+                </mwc-button>
+                ${this.addon.build
+                  ? html`
+                      <ha-call-api-button
+                        class="warning"
+                        .hass=${this.hass}
+                        path="hassio/addons/${this.addon.slug}/rebuild"
+                      >
+                        Rebuild
+                      </ha-call-api-button>
+                    `
+                  : ""}
+                ${this._computeIsRunning
+                  ? html`
+                      <ha-call-api-button
+                        class="warning"
+                        .hass=${this.hass}
+                        path="hassio/addons/${this.addon.slug}/restart"
```

Comment:
```suggestion
                        .path=`hassio/addons/${this.addon.slug}/restart`
```

Reply:
I had to revert all of these.
It broke the syntax.